WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -202,32 +209,41 @@ namespace Plugin {
 
     void WebKitBrowser::LoadFinished(const string& URL, int32_t code)
     {
-        TRACE(Trace::Information, (_T("LoadFinished: { \"url\": \"%s\", \"httpstatus\": %d}"), URL.c_str(), code));
```

Comment:
Agree, a Trace can never be bad :-)

Reply:
Yes, will add the status code to that Trace, was not removed intentionally but this happens when merging code that was changed in bit multiple repos quite drastically ;) 
Need to look into adding it to the notification to see if that would break compatibility.

Reply:
The "loaded:true" needs to be there not to break backwards compatibility.

Reply:
got it. thanks